src/modules/item/sagas.ts

```diff
@@ -451,6 +451,8 @@ export function* itemSaga(legacyBuilder: LegacyBuilderAPI, builder: BuilderClien
     yield put(saveItemRequest(newItem, {}))
     yield take(SAVE_ITEM_SUCCESS)
     yield put(closeModal('AddExistingItemModal'))
+    const address: string = yield select(getAddress)
```

Comment:
Would you mind moving the select as up as possible in this method? This is not he case, but re-dispatching actions might result in undesired behavior if the data is not retrieved before putting other actions.
Also, would you mind adding a test that exercises this?

Reply:
Thanks for your review. Sure, on it!